src/views/Hero/index.view.tsx

```diff
@@ -16,11 +16,17 @@ const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
     <div
       className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
     >
-      <PostcardStack pageName={pageName} />
-      <EmailSubscriptionForm />
-      Hello from HeroView
-      {title}
-      {description}
+      <div className="HeroView__container">
+        <div className="HeroView__visual">
+          <PostcardStack pageName={pageName} />
+        </div>
+        <div className="HeroView__textContainer">
+          <h1 className="HeroView__titleText">CruzHacks 2021 /</h1>
+          <h1 className="HeroView__titleText--secondary">{title}</h1>
+          <p className="HeroView__description">{description}</p>
```

Comment:
Would it also be advisable to change `h1` and `p` to `div` to remove any default styling that we might have to repetitively overwrite?

Reply:
Perhaps. Would it make the TSX file less readable if we don't use the heading and `p` tags for text? To me `div` tags are more for containers but that's a personal preference.

Reply:
We could also take advantage of the `h1, p, ...` elements by redeclaring their default styles in one of the base stylesheets!

Reply:
Another route would be to completely make text elements generic with `span`, although that's a pretty big compromise to HTML readability

Reply:
I think that's a great recommendation - we can forgo class names in exchange for general header formatting.

Reply:
@tim-nguyen-cs we want to stay organized with class names. We don't want to eliminate classmates. What we can do is add a general mixin for an `h1`, `p`, etc tags and include that mixin for each div.

Reply:
@tim-nguyen-cs I think @ilopezro wants to use class names though (https://github.com/CruzHacks/cruzhacks-2021-website/pull/9#discussion_r506986915), unless I'm misunderstanding your suggestion?

Reply:
@ilopezro I don't think we need to introduce unnecessary abstraction if we can avoid it— mixins are useful, but if we can select the HTML elements themselves why bother? I do agree that CSS classes are important & useful for granular styling, but if we can establish baseline style formatting let's do it.

Reply:
Especially in the `Hero` component, we don't really need the usage of mixins that much. We will be reusing hero component throughout our views. Let's stick with class names + defining everything at the Hero level since I don't see us reusing those outside of the hero component in order to avoid over abstraction! how does that sound y'all? I do still think we should change the `h1` tags into `div` tags and style the text under a `span` tag (or simply under the `div` tag), but that's just me.

Reply:
@ilopezro divs containing spans would get the job done, but it blows up the HTML file size if we introduce that redundancy. Plus it seems that using traditional HTML element references promotes readability (let's keep this source code interesting so it can be workshop material)

Reply:
Have we reached a consensus here?

Reply:
Just putting in my two cents, I am recommending these two approaches:

1. `div` tags with class names (alternatively also `span` tags for inline styling)
2. `h*` and `p` tags without class names (because the overlap would be redundant/superfluous)

This is just based on the development experience I've been encompassing, but let me know what the "law of the land" says.

Reply:
I like the idea of using `h*` and `p` tags without class names but the issue with that is that the CSS will get hard to follow and readability will be low. Also, by adding defaulting values for these tags as well, we are making a big assumption that all those tags will be rendered the same and I don't like that approach. I'd rather just define the `div` to what I want it to look like and using `span` or just plain text within the div itself.

Reply:
why not both & have classnames + simple elements? we can reset the default styles for established HTML elements to our own design pattern using base stylesheets & use granular classnames as well since CSS cascades

Reply:
I think that would defeat the purpose of having CSS files within each component and view if we're defining basic elements at the base level. I think let's stick with class names since this is the only place where I foresee we're using these specific styles. These are hero specific so I don't see why we should redefine these styles higher up in the tree. Let's keep these styles component specific w/ class names. 

Again, my suggestion here would be `div` and a `p` nested within the divs but `h1` also works for me. I usually don't use the header tags and nest a `span` or `p` within a `div`.

Reply:
here's a demo of how cascading base HTML elements into `div` containers can help us cut some abstraction: https://codepen.io/mocklee/pen/wvWzxzV

Reply:
Ok i see that. I personally don't like this approach because to reiterate, we'd be making a huge assumption about our tags if we move it further up the tree. The further down you to you have to set and unset props and that seem unnecessary. If there is code that we can refactor, we can should create mixins instead. We should make full use of the SCSS magic.

Reply:
updated the demo to demonstrate nested classnames. I'd say that refactors w/ adding granular classnames later on are tolerable, as long as there aren't too many distinct changes. 

also, it seems that mixins are being siloed to serve more as functions— [the Sass standard also has @extend available, which is syntactical sugar for short mixin declaration](https://sass-lang.com/documentation/at-rules/extend).

Reply:
Yeah I noticed the updated Codepen environment you've got up. If we go with pure classnames, there is no need for that mixin to unset the properties. That becomes redundant when you redefine them inside the another component. I wouldn't want to get too much technical debt if we over refactor and then later on find out we're doubling up on code because we've over refactored at the beginning. Let's stick with classnames for now. When there is a place to refactor, lets go for it!

Reply:
[SCSS psuedoclasses](https://css-tricks.com/the-sass-ampersand/) only run one level deep, don't overthink nesting! In general, CSS inheritance isn't hard; but nuanced— it depends on both local pseudoclasses & general DOM (HTML) tree structure. if classnames make it easier, we'll go with that.

if `%moreStyles` seems redundant, please demo an improvement!